fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDeviceHL.xtend

```diff
@@ -462,12 +464,16 @@ def dyn_attr(self, dev_list):
 # Run server
 # ----------

```

Comment:
Shouldn't this comment should be just before the `def main()` - that is the part that is running the server?

Also, the other usages of `cls.protectedAreaHL` are conditional, checking if the "protected regions" option is enabled.

Maybe something like:

```python
«IF cls.description.filestogenerate.toLowerCase.contains("protected regions")»«cls.protectedAreaHL("custom_functions")»«ENDIF»

# ----------
# Run server
# ----------

def main(args=None, **kwargs):
    """Main function of the «cls.name» module."""
    «IF cls.description.filestogenerate.toLowerCase.contains("protected regions")»«cls.protectedAreaHL("main", "return run((" + cls.name + ",), args=args, **kwargs)", false)»«ELSE»
    return run((«cls.name»,), args=args, **kwargs)«ENDIF»


if __name__ == '__main__':
    main()
```